Description: disable inclusion of accessory makefile needed for static build
Forwarded: https://github.com/samtools/samtools/issues/275
Author: Charles Plessy

--- a/Makefile
+++ b/Makefile
@@ -82,7 +82,7 @@
 
 # Adjust $(HTSDIR) to point to your top-level htslib directory
 HTSDIR = ../htslib
-include $(HTSDIR)/htslib.mk
+-include $(HTSDIR)/htslib.mk
 HTSLIB = $(HTSDIR)/libhts.a
 BGZIP  = $(HTSDIR)/bgzip
 
